### PR TITLE
Add font awesome loading spinner to CAPA problems

### PR DESCRIPTION
**Pull request by caesar2164**
loading spinner is automatically replaced by the content when AJAX problem_get returns

@stvstnfrd & @mondiaz

### CODE REVIEW

**Comment by mondiaz** [comment]
@caesar2164 based on the demo you gave of it just now, lgtm! 👍 thanks!

File: lms/templates/problem_ajax.html [hunk 1]
```diff
@@ -1 +1,6 @@
-<div id="problem_${element_id}" class="problems-wrapper" data-problem-id="${id}" data-url="${ajax_url}" data-progress_status="${progress_status}" data-progress_detail="${progress_detail}"></div>
+<div id="problem_${element_id}" class="problems-wrapper" data-problem-id="${id}" data-url="${ajax_url}" data-progress_status="${progress_status}" data-progress_detail="${progress_detail}">
+    <p class="loading-spinner">
+        <i class="fa fa-spinner fa-pulse fa-2x fa-fw"></i>
+        <span class="sr">Loading...</span>
```
Comment (cptvitamin) [inline lms/templates/problem_ajax.html]: consider ```&hellip;``` instead of ...

**Comment by caesar2164** [comment]
screen shot 2016-08-11 at 9 54 51 am

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**